### DISCUSSION TITLE
Continue to verification does nothing

### DISCUSSION THREAD

**Discussion by NovaDorium** [DSJAS/DSJAS#47] [Installation help] [answered]
I am using XAMPP and the setup went well until the install webpage where it tells me to copy the code and then continue to verification, problem is if I click continue, nothing happens. 
Even if I change the localhost directory to verification.php, it just brings me back to install.php
Is there something I'm missing?

**Comment by ejv2** [comment]
Can you upload you error log to a pastebin? It will be called "error_log" in a directory called "logs" in your XAMPP install directory (example path: "/opt/lampp/logs/error_log" or "C:\XAMPP\logs\error_log"). If this is an error in DSJAS, that will tell me what it is. Thanks.

**Comment by mickcott** [comment, marked as answer]
Any progress with this issue? Can report the same issue using the latest version beta v 0.1.2. Thanks.

**Reply by ejv2** [reply]
There are some known issues with using previous tags on the latest versions of PHP. In the near future, I will be relasing a patch release for some of the more recent changes. In the meantime, try downloading and building from source.

**Reply by mickcott** [reply]
Ethan, thanks for the prompt reply. Same problem building from source. It seems to be widespread, so bypassing the verification just leads on to db creation and subsequent failure for the same reason. The joys of PHP not being backwards compatible! Maybe I can try an earlier linux PHP version if you could tell me the last one that worked.

**Reply by mickcott** [reply]
"known issues with using previous tags on the latest versions of PHP"
Do you have a reference for this?

**Reply by ejv2** [reply]
According to the logs on my development box, the last version that I used for development was PHP 7.4.1, but any PHP 7 version should be pretty much safe. As for references, there have been numerous commits to master since the beta release fixing support for certain PHP versions.

It is possible that I may rewrite the entirety of the install code from scratch once I have a chance, as it seems to be the most buggy part of the codebase by far. It is not a coincedence that this is also the part of the code that has not been changed or rewritten in the most time.

I have opened an issue ticket for this problem at #56. For better debugging, can you upload an error log, as I asked the poster of this issue? This will make the finding of the issue much easier. Thanks for the help and apologies for the inconvenience.

**Reply by mickcott** [reply]
Thanks.

"For better debugging, can you upload an error log, as I asked the poster of this issue? This will make the finding of the issue much easier. Thanks for the help and apologies for the inconvenience."
Will do when I get a chance. FYI I tried  PHP 5.6.0 but that didn't help. Maybe I went too far back!

**Reply by ejv2** [reply]
No worries. I'm going to lock this thread now, so that we can discuss it in the new bug tracker, as this does sound like a bug in DSJAS to me. Thanks for the assistance. Bug report opened at #56.
